tests/d-cards.test.ts

```diff
@@ -1,20 +1,6 @@
 import { Scorer } from '../src/scorer';
-import { zeroPointBlueCard, zeroPointColourlessCard, zeroPointGreenCard, zeroPointPurpleCard, zeroPointRedCard } from './helpers';
+import { zeroPointBlueCard, zeroPointGreenCard, zeroPointPurpleCard, zeroPointRedCard } from './helpers';
 
-describe('Using DELICIOUS card', () => {
-  test('scores 0 if there is not a different colourless card', () => {
-    const scorer: Scorer = new Scorer([], [{'name': 'DELICIOUS'}]);
-    expect(scorer.scores()).toStrictEqual([0]);
-  });
-
-  test('scores 0 when there is a different colourless card it sc', () => {
-    const scorer: Scorer = new Scorer(
-      [],
-      [{'name': 'DELICIOUS'}, zeroPointColourlessCard()]
-    );
-    expect(scorer.scores()).toStrictEqual([4]);
-  });
-});
```

Comment:
??? why

Reply:
It's a basic card now man. 
No need to test that it scores 4